src/components/TableResize.js

```diff
@@ -94,21 +94,32 @@ class TableResize extends React.Component {
   };
 
   onResizeStart = (id, e) => {
-    this.setState({ isResize: true, id, startPosition: e.clientX });
+    this.setState({ isResize: true, id });
   };
 
   onResizeMove = (id, e) => {
-    const { startPosition, isResize, resizeCoords } = this.state;
-
+    const { isResize, resizeCoords } = this.state;
+    const fixedMinWidth = 100;
+    const idNumber = parseInt(id);
+    const finalCells = Object.entries(this.cellsRef);
+    if (idNumber >= finalCells.length) return;
     if (isResize) {
-      const leftPos = startPosition - (startPosition - e.clientX);
```

Comment:
Simple math x= a - (a -b )  is same as x= a-a +b which is same as x=b;

Reply:
Makes sense, thanks for explaining it to me. Since we're no longer using `startPosition`, it should be safe to remove it our state initialization as well.

Reply:
removed